trompace/mutations/musicgroup.py

```diff
@@ -145,3 +144,14 @@ def mutation_musicgroup_remove_exact_match_musicgroup(identifier_from: str, iden
     Returns: a mutation to remove the exactMatch relationship from the MusicGroup objects
     """
     return format_link_mutation("RemoveMusicGroupExactMatch", identifier_from, identifier_to)
+
+
+def mutation_add_musicgroup_member(identifier_from: str, identifier_to: str):
+    """Returns a mutation for linking a Person object with an MusicGroup object.
```

Comment:
add the name of the relation here (https://schema.org/member), because there could be many types of relationships between people and groups, and we need to say which one it is